src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain
+
+class Cars(registerCarNumber: Int) {
+
+    val cars: List<Car>
```

Comment:
아래와 같이 작성하여 init을 없앨 수 있을 것 같아요!

```suggestion
    val cars = registerCar(registerCarNumber)
```

Reply:
꼼꼼하지 못하게 코딩했네요 😅
좋은 피드백 감사드려요!